Sources/MapboxNavigation/Style.swift

```diff
@@ -49,6 +49,15 @@ open class Style: NSObject {
      */
     @objc open func apply() { }
     
+    /**
+     Applies the style based on provided `UITraitCollection`.
+     
+     - parameter traitCollection: `UITraitCollection` instance, properties of which
+     (e.g. `userInterfaceStyle`, `userInterfaceIdiom` etc), will be used for applying style.
+     */
+    @available(iOS 12.0, *)
+    open func apply(for traitCollection: UITraitCollection?) { }
```

Comment:
Let’s mention the addition of this method in the changelog.

Reply:
Decided not to add this public method. All styling configuration for CarPlay will happen in `DayStyle` (in case if CarPlay supports both light and dark styling).